docs/python/debugging.md

```diff
@@ -1,5 +1,5 @@
 ---
-Order: 6
+Order: 7
```

Comment:
Are these orders referring to which readme shows up first in a web page? or which readme gets listed first for a lists of all the other pages for Python extension in the wiki.

Reply:
This is the order in the table of contents on vscode-docs TOC